net/pfSense-pkg-haproxy-devel/files/usr/local/pkg/haproxy/haproxy.inc

```diff
@@ -41,6 +41,9 @@ $a_frontendmode['tcp'] = array('name' => "tcp", 'shortname' => "tcp");
 $a_frontendmode['health'] = array('name' => "health", 'shortname' => "health");
 */
 
+global $openssl111;
+$openssl111 = (OPENSSL_VERSION_NUMBER >= 269488128) ? true : false;
```

Comment:
The ternary operator here is redundant, if you assign the value of the test itself it has the same net result:
```php
$openssl111 = (OPENSSL_VERSION_NUMBER >= 269488128)
```

Reply:
Fixed